pkg/virt-handler/vm.go

```diff
@@ -3190,3 +3193,12 @@ func (d *VirtualMachineController) handleMigrationAbort(vmi *v1.VirtualMachineIn
 func isIOError(shouldUpdate, domainExists bool, domain *api.Domain) bool {
 	return shouldUpdate && domainExists && domain.Status.Status == api.Paused && domain.Status.Reason == api.ReasonPausedIOError
 }
+
+func (d *VirtualMachineController) hotplugVirtioInterfaces(vmi *v1.VirtualMachineInstance) error {
```

Comment:
We made an effort to move all network logic and have a single entry point for its setup.
This change has logic outside now, is there a reason not to have this moved into the netconf Setup ?

Except preserving a single entry point for network setup, we will also have less code which is not covered by tests. It will also remove the need to pass duplicate data (vmi & the pluggable networks).

Reply:
The duplication is in the data. Both the VMI is passed in and the partial list of networks which are part of this VMI.

I am hoping that you can pass the VMI to the network setup (in netconf.go) and do all the calculations there. Leaving (almost) no network logic externally.

Reply:
I am assuming this is the same issue as  https://github.com/kubevirt/kubevirt/pull/6852/commits/e6f5f793673e346c74357c4209e70655b8a5fdad#r1103802124; can I assume you'll tackle this as a follow-up ? @EdDev